scripts/openyurt-deployer/node.go

```diff
@@ -0,0 +1,1039 @@
+// MIT License
```

Comment:
this file has many long lines of code. Please break the long lines into multiple